shared/src/main/java/io/crate/concurrent/CompletableFutures.java

```diff
@@ -29,6 +29,9 @@
 
 public final class CompletableFutures {
 
+    public static final CompletableFuture<Boolean> COMPLETED_TRUE = completedFuture(true);
```

Comment:
Not sure if this is a good idea to define these as static. CompletableFuture has state